ksqldb-test-util/src/main/java/io/confluent/ksql/test/util/EmbeddedSingleNodeKafkaCluster.java

```diff
@@ -623,6 +624,7 @@ private void resetJaasConfig() {
     Configuration.setConfiguration(null);
   }
 
+  @SuppressWarnings("deprecation")
   private AdminClient adminClient() {
     final Map<String, Object> props = new HashMap<>(getClientProperties());
     props.put(AdminClientConfig.RETRIES_CONFIG, 5);
```

Comment:
Similar to above: are you trying to bound the test runtime in case of failure? If admin client you should set `default.api.timeout.ms` instead of `retries` for this case.